### ISSUE TITLE
Older solutions not working anymore: how could I know about them?

### ISSUE THREAD

**Issue by GiacomoSorbi** [Codewars/codewars.com#57]
I just finished [a kata](http://www.codewars.com/kata/5461c0e29726f66bb60010b1/) that apparently was worked a lot, so I was wondering if it was possible to get a warning of some sort about old/obsolete solutions to be refactored.

[While at it, I was also wondering if the code-warrior that completed the older versions of the kata are now losing something having solution no more working.]

**Comment by jhoffner** [comment]
There is an obsolete solutions list within your profile that you can look through. Are you asking for a notification about them? 

Also obsolete solutions do not lose you anything, except for any further up votes due to them being at the  bottom of the list.

**Comment by GiacomoSorbi** [comment]
Yeah, a notification would be good, as I see checking one's obsolete solutions rather unlikely.

I happened to see a few that were obsolete while reading past katas' solutions and found that being notified about them getting obsolete may be a good push to redo/improve past katas.

And thanks for answering on the latter part too :)

**Comment by javatlacati** [comment]
:+1:
